### PR TITLE
Exploratory Data Analysis section

### PR DESCRIPTION
**Pull request by kirsten-stallings**
Added the code and instruction to the exploratory data analysis section. Also added a bit about python scripting, procedural/OO. The purpose was to mainly explain the drastic difference in styles between the morning  and afternoon code (morning: one offs, lots of experimentation; afternoon: more structured, really only one way to make it work well)

### CODE REVIEW

File: machine_learning_tutorial.ipynb [hunk 1]
```diff
@@ -295,12 +291,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Normally, when we make models here at CallMiner, we focus heavily on the transcripts of calls. These transcripts are broken into turns that contain turn specific information, like who said it and when it was said. We don't quality call data to use as examples today. However, we do have Game of Thrones dialogue."
+    "Normally, when we make models here at CallMiner, we focus heavily on the transcripts of calls. These transcripts are broken into turns that contain turn specific information, like who said it and when it was said. Unfortunately, we don't quality call data to use as examples today. However, we do have Game of Thrones dialogue."
```
Comment (kirsten-stallings) [inline machine_learning_tutorial.ipynb]: somehow this commit didn't make it into the last pr.... weird... probably my fault.

Comment (jambran) [reply machine_learning_tutorial.ipynb]: Am I parsing this wrong...? Should it be "we don't have* quality call data..."?

File: machine_learning_tutorial.ipynb [hunk 2]
```diff
@@ -976,7 +2213,349 @@
...
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#plot a count plot of episodes per director\n",
```
Comment (jambran) [inline machine_learning_tutorial.ipynb]: pep8 asks for a space between the # and the comment. 
Couple other pep8 issues, but I know you were looking at auto-pep8ing later.

File: machine_learning_tutorial.ipynb [hunk 3]
```diff
@@ -976,7 +2213,349 @@
...
.../xte7BUUvG5I97j9YFWuKfX7mdFlpCKSYanoRpaFqkAKwDJ0xANTxklt/5yGtQDstV6NAAxo3EUgPjM7PVEASQYY1EASQDu8R7gXQDu7/LvBE5GkfHOW3bieycurUuIFyoNfPmHZ/G6q7fhpt2jESffN5XCuUJtXbXXKJfb2rl5Tw7z+QoulVQcPb2KuanUUDu3txrU8Oa3T+PoYgGpmIT3H5rDdCaOzz9+uuNjqE8jiAV+PBWzPI1eNMedoJyLiNHgnFuzp4Sf3xaeG9a0iH4iciVmADzHGPumn5JbzvkZGGGsUwDOAShwzr/ldCxjbC+AfQD+0Xazwhg7whg7zBj7KYHzDJz3vWYOWpPj//3uj6zbPv/4aVQbOt564+xASiZFmJtK4Vyhuk4vnMptu5lCO0gO7s5Bb3Ic/lEex84YneCj0E+xVSBPw2lcixdHzdH0M1kF73n1XhxdLOCJBe/pzxSeSgcwZ2kqHbM8DSd98F6wEuEC16Rh9nL4MVr2kuNhTYvoJyJX4qMwQlT/FT5Kbhlj4wDeDsMY7ACQYoz9osvhdwP4e865/V3cwzm/FUb3+Z8wxq50+Tv3msblyKVLG2U+e2H/TAa3zU3gwePn0dCaaDY5Pnt4Adduz+C1+6cC/Vu9MDeZQq3R6kBdqzWwXK5vinjqTWYy/MGnzqOs6rh6hHMwmxFaIP14GrWGjufOtUbTv+s2YyTLJx+e93xc1cwRpHxoxrsxmY4bnoZKsrjBhYHjPhLhJNYk+8qZGM8/OcRpEf1EpCO8Wz2NfwXgJc75Jc55A0a+4jUux96NttAU5/ys+e9JAP8M4GaX87uPc34r5/zWbduCr2S65/Y5XCqq+Pzjp/G9E5cwn6/grQd2jFSuwJp2ayaUT1nTbUffaMxkFWxLx/FPzxsFB/1Igl/O0ALmZxT402fXoHOO/eZ7MZmO464bZ/Gd5y5i2WOQZ6Vu5B+CmKk1lTKMRlHVoMjiwwJF8GNI6br5zWkAxvdvq5XbAt3paRQF9TROAXgVYyzJjGzxGwE86/B3XgZgHMAjttvGGWNx8/9TAA4BeMbfSwsG0sL+3JFFfOaRBeSSMn7suulhnIor1KtBZbYticnNkYS7aXcOqtZEQpYC6QQPadFaIMXDU9Zoepvm9vsPzaHa0PHJh19yfRzpXgRRqTaRjmGt2kCx1jDFqoILT8Wj4
```
Comment (jambran) [inline machine_learning_tutorial.ipynb]: maybe talk a bit about what makes clean data clean (and dirty data dirty)

File: machine_learning_tutorial.ipynb (hunk 1 above)
Comment (kirsten-stallings) [reply machine_learning_tutorial.ipynb]: omg. It should. I don't don't know how the have keeps getting lost. I've changed it a billion times 😭

File: machine_learning_tutorial.ipynb (hunk 3 above)
Comment (kirsten-stallings) [reply machine_learning_tutorial.ipynb]: good idea. Added it in.

**Pull request merged**